hs_discover/src/components/Resources.vue

```diff
@@ -0,0 +1,752 @@
+<template>
```

Comment:
I'm surprised this component is as a big as it is, I think it makes sense to break it up into smaller components.  For example maps could be broken out.  I could even see each of the filters being broken out, it's a bit intimidating with almost everything heaped into this file.

Reply:
This used to be smaller, but as we pulled more into the reactivity, it grew. Many of the requirements will change during Enhancements phase, so at that time, the rewrite will likely spawn smaller components. However, this is far easier to read and debug than the global mess that existed before. Tracing implied Global JavaScript contexts, without encapsulation is impossible. I'm currently in the process of cleaning out base template to address this and the impact to JS and CSS,.

Reply:
So I looked at this again, it's an ok size for a vue file. It's broken into three distinct sections, per the file specification and even though it contains three sections, it's still shorter in line length than some code-only pages we have/had in JavaScript with global scopes and weak object oriented code. Don't forget maps.js has the core maps code and is instantiated and acted on. Due to valuable stakeholder feedback during this bugfix phase and the planned updates as a part of Discover Enhancement work, it's best to keep everything in one file to shift elements in the HTML view as well as interact across the JavaScript. As a part of Enhancements, it may or may not be appropriate/possible to split out additional sub components in Vue. I can cover this with you later when I go over how child components exchange data with parent components.